arkouda/numeric.py

```diff
@@ -1003,7 +1004,9 @@ def arctan2(
     TypeError
         Raised if the parameter is not a pdarray
     """
-    if not all(isSupportedNumber(arg) or isinstance(arg, pdarray) for arg in [num, denom]):
+    if not all(
+        isSupportedNumber(arg) or isinstance(arg, pdarray) for arg in [num, denom]
+    ):
```

Comment:
I think a lot of these reformatting changes are due to running `black` without the `--line-length 105` flag

Reply:
amanda already pointed this out!